tests/tests.js

```diff
@@ -326,104 +312,311 @@ QUnit.test("Initialize CheckIt test", function( assert ) {
+    assert.equal(this.$calendarDiv.find('.week').length, 6);
     
-    var fixture = this.fixture;
-    var testmonth = this.testmonth;
+    this.checkit.removeEmptyWeeks(this.calendar, this.$calendarDiv);
+    assert.equal(this.$calendarDiv.find('.week').length, 5);
     
-    fixture.find('#calendarDiv').append(`<div class="monthframe" id="${testmonth.monthId}"></div>`);
-    fixture.find(`#${testmonth.monthId}`).load('template.html', function() {
-        
-        // Test that template was added as a child
-        assert.equal($(this).children().attr('id'), 'template');
-        assert.equal($(this).children().length, 1, "Number of children of monthframe div");
-        
-        testmonth.fillMonthDiv();
-        // May need to modify the below when I switch to using fully filled months. but possibly not depending on
-        // if I still use the class actualDay
-        assert.equal($(this).find(".actualDay").first().children('.cell').attr('id'), 20170214, "Check start day of the month");
-        
-        $(this).find('.cell').each( function(index) {
-            assert.equal($(this).attr('id'), '201702' + (index + 14), "Checking id's of .cell divs");
-        });
-        
-        assert.equal($(this).find(".month-year").text(), "February 2017");
-        assert.notOk(jQuery.isEmptyObject(testmonth.dayIndex), "dayIndex object is not empty");
-        assert.equal(Object.keys(testmonth.dayIndex).length, 15, "Asserting number of keys in dayIndex object");
-        
-        assert.equal($(this).find('.actualDay').length, 15, "Checking the number of .actualDay td elements");
-        
-        assert.equal($(this).find('.cell').length, 15, "Correct number of .cell div elements");
-        
-        assert.equal($(this).find('.nill').length, 27, "Correct number of .nill div elements");
-        
-        assert.equal($(this).find('.cell').children().length, 30, "Total number of children of each .cell div");
-        assert.ok($(this).find('.daynumber').text(), ".daynumber divs have text in them");
-        
-        $(this).find('.daynumber').each( function(index) {
-            assert.equal($(this).html(), index + 14, "Correct daynumbers");
-            });
-            
-        $(this).find('.cell').each(function (index) {
-            assert.ok($(this).find('.element').hasClass("hidden"), "Check that .element divs have a hidden class");
-        });
-        
-        done();
-    })
-   
 });
 
-QUnit.test("removeEmptyWeeks test", function( assert ) {
-    // This test will need to be modified if an entire month is included 
-    // with grayed out days for inactive days
-    assert.expect(2);
+QUnit.test("findCurrentDay test", function( assert ) {
+    assert.expect(3);
     
-    var done = assert.async();
-    var fixture = this.fixture;
-    var testmonth = this.testmonth;
+    var currentDay = "20170214";
+    assert.equal(this.$calendarDiv.find('#20170214').attr('class'), 'cell');
+    this.checkit.findCurrentDay();
```

Comment:
What does this function do? The name suggests that it's going to e.g. locate the dom element corresponding to today and return it or something.

When you write functions that are going to mutate state, it's important to make sure that it's obvious that that's what's happening.